pkg/querier/tenantfederation/merge_queryable_test.go

```diff
@@ -18,6 +20,7 @@ import (
 
 	"github.com/cortexproject/cortex/pkg/querier/series"
 	"github.com/cortexproject/cortex/pkg/tenant"
+	sl "github.com/cortexproject/cortex/pkg/util/spanlogger"
```

Comment:
I think I wouldn't rename the import necessarily. What is your thinking there?

Reply:
done